Logger/ConsoleLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Logger;
+internal class ConsoleLogger : BaseLogger
+{
+    public override void Log(LogLevel logLevel, string message)
+    {
+        string msgOut = $"[{DateTime.Now} {CallingClass}";
+        switch (logLevel)
+        {
+            case LogLevel.Error:
+                msgOut += "Error";
+                break;
+            case LogLevel.Warning:
+                msgOut += "Warning";
+                break;
+            case LogLevel.Information:
+                msgOut += "Information";
+                break;
+            case LogLevel.Debug:
+                msgOut += "Debug";
+                break;
+        }
+
+        Console.WriteLine(msgOut + "] " + message);

```

Comment:
Consider using string interpolation here
```suggestion
        Console.WriteLine($"{msgOut}] {message}");

```

Reply:
Additionally, you may consider getting the log level and setting that to a variable and then building your string message, to improve readability, rather than having the switch statement in the middle and appending to your string within your switch.